### ISSUE TITLE
You forgot to sell major advantage of Sublime Text

### ISSUE THREAD

**Issue by loopywittgenstein** [aviaryan/SublimeNotebook#16]
is code formatting and Display. No really - CintaNotes and the rest do not recognize Python, .JavaScript, HTML etc. Use to Store lots of Code snip its. Took a while to sink in - Duh Oh I get it.

**Comment by aviaryan** [comment]
You are right. I should probably add that in the README and the setup docs as well.
I guess I missed it because I thought it's too obvious.
